### ISSUE TITLE
require('.') broken

### ISSUE THREAD

**Issue by sebv** [appium/ios-webview-app#2]
https://github.com/appium/ios-webview-app/blob/master/Gruntfile.js#L3

There is no index.js.

**Comment by Jonahss** [comment]
in package.json, `main` is set to `apps.json`.
Calling `require('.') returns
```
[ 'build/Release-iphoneos/WebViewApp-iphoneos.app',
  'build/Release-iphonesimulator/WebViewApp-iphonesimulator.app' ]
```

Which part of this is broken?

**Comment by Jonahss** [comment]
fixed in v1.0.1
